content/en/docs/Getting started/_index.md

```diff
@@ -88,31 +88,63 @@ To copy the example site:
 
 1.  Make a local working copy of the example site directly using `git clone`:
 
-    ```bash
-    git clone https://github.com/google/docsy-example.git
-    ```
-1. In GitHub, create a [new empty repo](https://help.github.com/en/articles/create-a-repo) for your site with your chosen repo name. For clarity you may also want to rename the root directory (`docsy-example`) of your local working copy to match, though everything will still work even if you don't.
-
-1. In your site root directory, change your local copy's `origin` to your new repo (otherwise you'll be trying to push changes to the Docsy example site rather than to your own site):
+     ```shell
+     git clone https://github.com/google/docsy-example.git
+     ```
+    
+1. Switch to the root of the cloned directory, for example:
 
-    ```bash
-    $ git remote remove origin
-    $ git remote add origin https://github.com/my/example.git
-    $ git push -u origin master
-    ``` 
+    ```shell
+    cd docsy-example
+    ```
 
 1. Get local copies of the project submodules so you can build and run your site locally:
 
-    ```bash
-    $ git submodule update --init --recursive
+    ```shell
+    git submodule update --init --recursive
     ```
-1.  Switch to your site root directory, and build your site:
     
+1. Build your site:
+    
+    ```shell
+    hugo server
     ```
-    $ hugo server
-    ```
-1.  Preview your site: http://localhost:1313/.
     
+1. Preview your site in your browser at: http://localhost:1313/
+
+    Tip: Use `Ctrl + c` to stop the Hugo server.
+
+1. Now that you have a local copy of the site running, you can set your `origin` remote and then 
+   push the site to your repository. For example:
+
+   1. If you need to create a remote repository, go to GitHub and create a 
+      [new empty repo](https://help.github.com/en/articles/create-a-repo) 
+      for your site with your chosen repo name. For clarity you may also want to rename the root 
+      directory (`docsy-example`) of your local working copy to match, though everything will still 
+      work even if you don't.
+
+   1. Ensure that your 
+      [remote (`origin`) is configured](https://help.github.com/en/articles/configuring-a-remote-for-a-fork):
+   
+       1. If you are not sure if you have a remote configured, run:
+      
+           ```shell
+           git remote
+           ```
+      
+       1. From your site's root directory, change your local copy's `origin` to your new repo (otherwise 
+          you'll be trying to push changes to the Docsy example site rather than to your own site):
+
+           ```shell
+           git remote remove origin
```

Comment:
i tried a few times and my `origin` was never pre set to docsy so this was slightly confusing to me (ie there was no way that i could accidentally try to push to the docsy repo)

Reply:
Are you sure? I literally just tried this and the origin is pre-set to the site you just cloned:

$ git clone https://github.com/google/docsy-example.git
$ cd docsy-example
$ git remote get-url origin
https://github.com/google/docsy-example.git

It's why you need to reset it once you've cloned it.

Reply:
as we chatted in the other PR, i was running the incorrect command and getting a false result (`git remote` vs `git remote get-url origin`). thanks @LisaFC!